lib/tasks/peer_review.rake

```diff
@@ -8,8 +8,17 @@ namespace :db do
     a1 = Assignment.find_by(short_identifier: 'A1')
     a1.update(has_peer_review: true)  # Creates 'a1pr' via callback.
 
+    # The group names must not be closed, doing a1pr.clone_groupings_from(a1.id)
```

Comment:
I think you mean "cloned," but I also think that this is not a problem if you resolve my above comment.